maven/plugins/com.liferay.ide.maven.core/plugin.xml

```diff
@@ -43,6 +43,16 @@
       </liferayProjectProvider>
    </extension>
    <extension
+         point="com.liferay.ide.core.liferayProjectProviders">
```

Comment:
I felt confuse that how to deal with gradleProvider. Assume I merge mavenMoudleProjectProvider function to mavenProjectProvider, then means the gradleProjectProvider also need to extends NewLiferayProjectProvider, right?